examples/jax/quickstart.ipynb

```diff
@@ -0,0 +1,637 @@
+        "@dataclass\n",
+        "class AConfig:\n",
+        "    a_lambda: float = 0.95\n",
+        "\n",
+        "\n",
+        "class simpler_advantage_estimate(GAE):\n",
+        "    def __init__(\n",
+        "        self,\n",
+        "        config: AConfig = AConfig(),\n",
+        "    ):\n",
+        "        self.config = config\n",
+        "\n",
+        "    def on_training_utility_fns(self, trainer: SystemTrainer) -> None:\n",
+        "        def simpler_advantage_estimate(\n",
+        "            rewards: jnp.ndarray, discounts: jnp.ndarray, values: jnp.ndarray\n",
+        "        ) -> Tuple[jnp.ndarray, jnp.ndarray]:\n",
+        "\n",
+        "\n",
+        "            # Instead of using the GAE we use a simpler one step\n",
+        "            # advantage estimate.\n",
+        "\n",
+        "            # Pad the rewards so that rewards at the end can also be calculated.\n",
+        "            zeros_mask = jnp.zeros(shape=rewards.shape)\n",
+        "            padded_rewards = jnp.concatenate([rewards, zeros_mask], axis=0)\n",
+        "            cum_rewards = rewards.copy()\n",
+        "            seq_len = len(rewards)\n",
+        "            for i in range(1, seq_len):\n",
+        "                cum_rewards+=padded_rewards[i : i +seq_len]\\\n",
+        "                              *jnp.power(self.config.a_lambda, i)\n",
+        "\n",
+        "            # Calculate the advantage estimate.\n",
+        "            advantages = cum_rewards[:-1] - values[:-1]\n",
+        "            \n",
+        "            # Stop gradients from flowing through the advantage estimate.\n",
+        "            advantages = jax.lax.stop_gradient(advantages)\n",
+        "\n",
+        "            # Set the target values and stop gradients from flowing backwards\n",
+        "            # through the target values.\n",
+        "            target_values = cum_rewards[:-1]\n",
+        "            target_values = jax.lax.stop_gradient(target_values)\n",
+        "\n",
+        "            return advantages, target_values\n",
+        "\n",
+        "        trainer.store.gae_fn = simpler_advantage_estimate\n",
+        "\n",
+        "    @staticmethod\n",
+        "    def name() -> str:\n",
+        "        return \"gae_fn\"\n",
+        "\n",
+        "    @staticmethod\n",
+        "    def config_class() -> Optional[Callable]:\n",
+        "        return AConfig"
+      ],
+      "metadata": {
+        "id": "A0HOxOnVmq93"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Now that we have this new component we can add it to the PPO system using `system.update`. You can retrain the system by executing the cell below. Once training is complete you can again run steps 4 and 5 (above) to view the training results and gameplay footage. Does this system perform better or worse than the previous system? \n",
```

Comment:
A bit misleading to say "new component" since we use `update()`. We should explain somewhere (as mentioned above, probably when we say `system.update(MonitorExecutorEnvironmentLoop)`) that `update` is because we have the same name/ extending an existing component and that if it is an entirely new component, we use `add`. Ideally we would have `system.add` in this part of the notebook but we can revisit this later

Reply:
Great idea. I updated the notebook!